variables.tf

```diff
@@ -10,6 +10,36 @@ variable "instance_name" {
   default     = ""
 }
 
+variable "ami_id" {
```

Comment:
we should remove this variable.

Reply:
most of environment use custom AMIs to ensure security agents are installed, so this will be needed. we just need to provide a requirement to use Amazon Linux based image.

Reply:
I agree with Raj's reasoning on this... probably we are going to need it.